shells/default/default.nix

```diff
@@ -3,6 +3,7 @@
   commitizen,
   hivemind,
   just,
+  yq,
```

Comment:
This was in `flox-cli-tests` but not the devshell. I found it useful for manipulating the manifest and interactively testing how it was being used in the tests.